### ISSUE TITLE
Invalid discord link

### ISSUE THREAD

**Issue by smalls89** [spacedriveapp/spacedrive#2790]
### Check for other issues

- [X] I've checked for other issues, and none exist.

### Issue description

Discord link (https://discord.com/invite/gTaF2Z44f5) is invalid
![image](https://github.com/user-attachments/assets/53728d2b-d0ab-4583-80c6-96c16a9d2f51)

### Steps to reproduce

1. click the discord link
2. be told it is invalid

### Platform and versions

- OS: all
- Spacedrive: na

### Code of Conduct

- [X] I agree to follow the Code of Conduct.

**Comment by iLynxcat** [comment]
Good catch, I see we missed updating this in a few places. It's a surprise nobody reported it until you did just now!

**Comment by iLynxcat** [comment]
Huh, looking into this deeper (I didn't give it the proper look as I didn't have time earlier) and that link looks to be all good everywhere I can find it. Can you send me the exact link for where you found this link and tell me which exact spot on the page it is?

**Comment by smalls89** [comment]
The link exists on the .com page as well as the GitHub readme page. Oddly, it worked just fine on mobile, and said nope on PC/chrome. As for what the ultimate issue is, I'm fine with saying the browser is the issue.
